packages/fluentui/react-northstar/src/components/Tree/TreeTitle.tsx

```diff
@@ -101,21 +142,51 @@ const TreeTitle: React.FC<WithAsProp<TreeTitleProps>> & FluentComponentStaticPro
 
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(TreeTitle.handledProps, props);
-
   const handleClick = e => {
     _.invoke(props, 'onClick', e, props);
   };
 
+  const selectIndicator = Box.create(
+    customSelectIndicator
```

Comment:
Where are we actually invoking the `customSelectIndicator` fn/component?

Reply:
So what is happening here is that if `customSelectIndicator` is defined we pass the props expected in the custom component and use the `customSelectIndicator` in the `as` prop bellow.

Reply:
Removed the customSelectIndicator